### PR TITLE
Support alternate fzy implementations that may deliver multiple matches

### PR DESCRIPTION
**Pull request by ohjames**
1. Can use alternative binaries to `fzy`.
2. Supports binaries that can deliver multiple matches, e.g. some fuzzy finders have a keybinding to select all matches instead of a single match.

### CODE REVIEW

File: plugin/neovim-fuzzy.vim [hunk 1]
```diff
@@ -208,16 +212,18 @@ function! s:fuzzy(choices, opts) abort
...
-      if has_key(file, 'lnum')
-        silent execute file.lnum
-        normal! zz
-      endif
+    let results = readfile(self.outputs)
+    if !empty(results)
+      for result in results
+        let file = self.handler([result])
+        exe 'lcd' self.root
+        silent execute g:fuzzy_opencmd fnameescape(expand(file.name))
```
Comment (ohjames) [inline plugin/neovim-fuzzy.vim]: When `g:fuzzy_opencmd` is `tab drop` then it's great to be able to open multiple matches like this, when `opencmd` is set to `:edit` it won't be so useful.

**Comment by ohjames** [comment]
These PRs: https://github.com/jhawthorn/fzy/pull/40 and https://github.com/jhawthorn/fzy/pull/32 also add support for multiple matches to `fzy`, this patch offers future support for them.

**Comment by cloudhead** [comment]
Cool. Unfortunately since I merged the previous PR for piping directly, this one now has conflicts. If you fix them I'll merge it.

**Comment by ohjames** [comment]
Cool I'll rebase and fix after #27 gets merged.

**Comment by ohjames** [comment]
@cloudhead Since #27 is hanging around and you reverted the aforementioned PR maybe I can rebase this and get it merged?

**Comment by ohjames** [comment]
My goal is to use this plugin with `fzu` so I can get configurable keybindings and multiple file selection.

**Comment by cloudhead** [comment]
Yeah sure thing. What is fzu?

**Comment by ohjames** [comment]
I forked fzy to add configurable keybindings and multiple file selection support. The fzy maintainer hasn't replied to a PR in months but he does occasional add some small work so it seems he's not interested in accepting contributions.

**Comment by ohjames** [comment]
Thanks!

**Pull request merged**